src/fixate/drivers/dmm/keithley_6500.py

```diff
@@ -0,0 +1,343 @@
+from threading import Lock
+from pyvisa import constants
+from fixate.core.exceptions import InstrumentError, ParameterError
+from fixate.core.common import mode_builder, deprecated
+from fixate.drivers.dmm.helper import DMM
+import time
+
+
+class Keithley6500(DMM):
+    REGEX_ID = "KEITHLEY INSTRUMENTS,MODEL DMM6500"
+    INSTR_TYPE = "VISA"
+
+    def __init__(self, instrument, *args, **kwargs):
+        self.instrument = instrument
+        instrument.rtscts = 1
+        self.lock = Lock()
+        self._display = "ON100"
+        # del self.instrument.timeout
+        self.instrument.timeout = 10000
+        self.instrument.query_delay = 0.3  # Stop DMM crash
+        self.instrument.delay = 0  # Stop DMM crash
+        self.is_connected = True
+        self.reset()
+        self._manual_trigger = False
+        self._samples = 1
+        self._CLEAN_UP_FLAG = False
```

Comment:
I guess I kept a lot of these flags just in case some peoples code go looking for them? In the end I guess its a problem with not having a proper base class.

Yes, I neglected the analog and digital filters because they aren't used, and I don't really see the benefit for having them. The Keithley does not have an analog filter either.

Reply:
The atgrep tool doesn't return any hits for _CLEAN_UP_FLAG, _ANALOG_FLAG or _DIGITAL_FLAG. I'm not sure why any of them exist. I'd be inclined to remove them, we can always add them back in later if there ever is a need for them.
For the stuff in the helper, I'm not sure which side of the fence that I stand on for requiring all of the functions in the helper file to be implemented. We definitely need any method in a driver to appear in the helper because of how fixate works, but it's possible that we might have another DMM that doesn't support other methods that are declared in the helper. So we might need to think about how we deal with the not-implemented errors that will occur if someone uses a method that's not supported on a particular instrument.

Reply:
Update, the fluke driver sets `_CLEAN_UP_FLAG` in `error_cleanup()`, but this isn't done in this driver.

Reply:
Clean up flags are removed, as well as unused clean up functions